src/main/kotlin/no/nav/familie/ks/sak/api/BehandlingController.kt

```diff
@@ -57,6 +58,17 @@ class BehandlingController(
         return ResponseEntity.ok(Ressurs.success(behandlingService.lagBehandlingRespons(behandlingId = behandling.id)))
     }
 
+    @GetMapping(path = ["fagsak/{fagsakId}"])
```

Comment:
Tidenes nitpick, men kan du legge til en prefiksed / her? 
```suggestion
    @GetMapping(path = ["/fagsak/{fagsakId}"])
```

Det har ingen ting å si fordi det legges på uansett hvis vi ikke gjør det, men med tanke på at alle andre endepunkter i denne filen har det så er det greit å fortsette samme mønster.

Reply:
Kjempebra med nitpick! Jeg burde ha sjekket selv for å være konsekvent med annet i filen, takk som sier ifra.<img src="https://media2.giphy.com/media/v1.Y2lkPWJkM2VhNTdlMWh5YjgzeXN4bWdyaW41amM2YXQ1N2tvbXNsMnY0N216ZHVqYmUyYiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/AXdGgQfbeng2x7cX8i/giphy.gif"/>